cmd/snap-repair/runner.go

```diff
@@ -220,9 +217,6 @@ func (r *Repair) Run() error {
 	// read from the status-pipe, however report the error
 	if scriptErr != nil {
 		scriptErr = fmt.Errorf("repair %s revision %d failed: %s", r, r.Revision(), scriptErr)
-		if err := r.errtrackerReport(scriptErr, status, logPath); err != nil {
```

Comment:
it would be good to replace some of the places that were doing this with:

// TODO: telemetry about errors here

Reply:
Done!